dts/arm/st/f3/stm32f3.dtsi

```diff
@@ -11,6 +11,10 @@
 #include <dt-bindings/gpio/gpio.h>
 
 / {
+	aliases {
```

Comment:
I don't think zephyr specific aliases should be put under dts/ in .dtsi files
These should be done in boards .dts, similarly to other aliases

Reply:
@erwango Thank you for reviewing.

Why don’t the aliases belong in dtsi? That’s how many other (if not all other) SoCs handle it.

Reply:
My view is that aliases are zephyr 'upstream' specific, they are used to make zephyr supported boards to fit with samples and tests.
.dtsi files are made to be used by upstream boards but also out of tree boards for out of tree applications, so they should remain application agnostic.

Reply:
Not sure I understand your argument, but if you would rather I add the alias to all boards using the stm32's, I can do that.

I feel we need a common policy on this instead of having different policies depending on the vendor of the SoC. /cc @galak